Source/LibationWinForms/grid/ProductsGrid.cs

```diff
@@ -37,6 +38,7 @@ namespace LibationWinForms
 
 	public partial class ProductsGrid : UserControl
 	{
+		internal event EventHandler<GridEntry> LiberateClicked;
```

Comment:
Yeah, both will be used eventually (or something like them), but I can remove them for now.

My vision for the overall architecture is to go full MVVM, with GridEntry being the viewmodel.  UI elements should only interact with the database through GridEntry.  That may not be possible and I understand if you have different priorities, but doing it that way would allow for much greater separation of concerns and looser coupling.